src/api/json/catalog.json

```diff
@@ -944,6 +944,12 @@
       "fileMatch": ["cumulusci.yml"],
       "url": "https://raw.githubusercontent.com/SFDO-Tooling/CumulusCI/main/cumulusci/schema/cumulusci.jsonschema.json"
     },
+    {
+      "name": "DataYoga Job",
+      "description": "DataYoga job configurations",
+      "fileMatch": ["**/jobs/*.yaml", "**/jobs/*.yml"],
```

Comment:
This seems to be fairly generic and could cause false positive file matches. @hyperupcall what do you think?

Reply:
@madskristensen I understand your concern. What if we have a `.dy` in our root folder, is this something we can rely on? So all yaml files within this tree will be used this jsonschema? 
What do you do in similar cases?
Please, advise.

Thanks!

Reply:
Yes, that would work and make it a lot more specific

Reply:
Yeah, I think the fileMatch is a bit too generic. Like you mention, perhaps something like `.dy/jobs/*.yml` could work?

But if I interpret `.dy` directory is adjacent to `jobs` and not a parent, then I'm afraid there is no way to express that relationship in these globs

Reply:
Yes, I suspected that too @hyperupcall, thanks for your input. I'll change the fileMatch to expect `.dy.yaml` and `.dy.yml` patterns instead, this is also legit and will solve @madskristensen 's concern.

Reply:
That works - thanks.